libvirt/tests/src/virtual_device/vsock.py

```diff
@@ -244,6 +287,28 @@ def managedsave_restore():
                 else:
                     result = virsh.start(vm_name, debug=True)
                     utils_test.libvirt.check_exit_status(result, expect_error=False)
+        elif guest_user:
+            if not libvirt_version.version_compare(10, 4, 0):
+                test.cancel("SSH proxy for vsock is not supported in this version.")
+            else:
+                if not utils_package.package_install(["libvirt-ssh-proxy"]):
+                    test.cancel("Failed to install libvirt-ssh-proxy on host")
+
+                vmxml.add_device(vsock_dev)
+                vmxml.sync()
+                logging.debug(vmxml)
+                virsh.start(vm_name, ignore_status=False)
+                vm_ip = vm.wait_for_get_address(0, timeout=240)
+                vm.prepare_guest_agent()
+                add_guest_user(guest_user, guest_user_passwd)
+                ssh_key.setup_ssh_key(vm_ip, guest_user, guest_user_passwd)
+
+                if int(check_systemd_version()) < 256:
+                    forward_vsock_to_sshd()
+
+                ssh_proxy_cmd = "ssh %s@qemu/%s hostname" % (guest_user, vm_name)
+                if process.run(ssh_proxy_cmd, shell=True).exit_status != 0:
```

Comment:
```suggestion
                if process.run(ssh_proxy_cmd, shell=True, ignore_status=True).exit_status != 0:
```
Or it will raise CmdError if the command failed